docs/getting-started.md

```diff
@@ -21,6 +21,26 @@ The Client SDK is a set of user-facing programs to interact with Ledger and Audi
 ScalarDL (Ledger and Auditor) abstracts data as a set of assets, where each asset is composed of the history of a record identified by a key called `asset_id` and a historical version number called `age`.
 In this document, you will create a very simple application to manage an asset's status using ScalarDL Client SDK.
 
+## Install a JDK
+
+Because ScalarDL is written in Java, using Java is one of the easist ways to build your ScalarDL applications.
+In such a case, you must have one of the following Java Development Kits (JDKs) installed in your environment:
+
+- [Oracle JDK](https://www.oracle.com/java/technologies/downloads/) LTS version (8, 11, or 17)
+- [OpenJDK](https://openjdk.org/install/) LTS version (8, 11, or 17)
+
+Also, since ScalarDL is built with JDK 8, you must use JDK 8 to build your contracts.
+If you use JDK versions other than 8, specify `-release 8` option for `javac` or define what toolchain to use in your Gradle and Maven configurations.
```

Comment:
```suggestion
If you use JDK versions other than 8, specify `--release 8` option for `javac` or define what toolchain to use in your Gradle and Maven configurations.
```
according to the description
```
$ javac -help | grep -A 1 '\--release'
  :
--
  --release <release>
        Compile for a specific VM version. Supported targets: 6, 7, 8, 9, 10, 11
```

Reply:
Oops, thanks!

Reply:
My suggestion below includes [@/komamitsu suggested change](https://github.com/scalar-labs/scalardl/pull/54/files#r1483902153).

```suggestion
If you use a version other than JDK 8, specify the `--release 8` option for `javac` or define which toolchain to use in your Gradle and Maven configurations.
```